ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -81,7 +90,13 @@ interface NewGitHubAppRepoCredsParams {
 export enum ConnectionMethod {
     SSH = 'via SSH',
     HTTPS = 'via HTTPS',
-    GITHUBAPP = 'via GitHub App'
+    GITHUBAPP = 'via GitHub App',
+    GOOGLECLOUD = 'via Google Cloud'
+}
+
+interface NewGoogleCloudSourceRepoCredsParams {
```

Comment:
All the UI changes look good - very similar to the other connection type and credentials.  The only thing I can nitpick on is just to move this interface above the enum so all the interfaces are 'together'.

Reply:
updated!